lib/assign-properties.js

```diff
@@ -37,7 +45,7 @@ function giveMethodSuper(superclass, name, fn) {
 
 var sourceAvailable = (function() {
   return this;
-}).toString().indexOf('return this;') > -1;
+}).toString().indexOf('return this') > -1;
```

Comment:
Some minification/obfuscation will remove the semicolon above this line and all hell breaks loose.